### PR TITLE
Walkthrough fixes after testing

### PR DESCRIPTION
**Pull request by minsa110**
For #8799 and #8800
- Fixed images not showing by moving them to the resources folder
- Fixed command for walking users through selecting a kernel

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -68,6 +68,7 @@
         "onCommand:jupyter.selectNativeJupyterUriFromToolBar",
         "onCommand:jupyter.clearSavedJupyterUris",
         "onCommand:jupyter.filterKernels",
+        "onWalkthrough:jupyterWelcome",
```
Comment (rchiodo) [inline package.json:71]: Something is off here. Did you start from the latest version of main? Meaning you did a 
- git checkout main
- git pull/fetch
- git checkout -b walkthrough

It looks like your baseline is from before your first change.

File: package.json [hunk 2]
```diff
@@ -80,6 +81,61 @@
...
+                            "image": "resources/walkthroughs/dataviewerdark.gif",
+                            "altText": "Opening data viewer"
+                        }
+                    },
+                    {
+                        "id": "jupyter.dataScienceLearnMore",
+                        "title": "Find out more!",
+                        "description": "🎨 Explore all the features the Jupyter extension has to offer by looking for \"Jupyter\" in the [Command Palette](command:workbench.action.showCommands). \n 📈 Learn more about getting started with data science in [Python](command:workbench.action.openWalkthrough?%7B%22category%22%3A%22ms-python.python%23pythonDataScienceWelcome%22%2C%22step%22%3A%22ms-python.python%23python.createNewNotebook%22%7D) by installing the Python extension ([search Python extension](command:workbench.extensions.search?\"ms-python.python\")). \n 📃 Find out more features in our [Tutorials](https://code.visualstudio.com/docs/datascience/jupyter-notebooks).  \n[Learn more](https://aka.ms/AAdar6q)",
+                        "media": {
+                            "image": "resources/walkthroughs/interactive.gif",
```
Comment (rchiodo) [inline package.json:132]: I would expect to only see changes on these lines.

File: package.json (hunk 1 above)
Comment (rchiodo) [reply package.json:71]: If you do a 
- git fetch origin
- git rebase origin/main

That should fix the problem (after you fixup conflicts)

Comment (minsa110) [reply package.json:71]: No, I used the branch from my previous commit.... 🤦🏻‍♀️ will close this PR and submit a new one

**Pull request closed without merging**